src/ripple/rpc/impl/LookupLedger.cpp

```diff
@@ -34,15 +35,16 @@ bool isValidatedOld ()
         Tuning::maxValidatedLedgerAge;
 }
 
-Status ledgerFromRequest (
-    Json::Value const& params,
-    Ledger::pointer& ledger,
-    NetworkOPs& netOps)
+template <class T>
+Status ledgerFromRequest (T& ledger, Context& context)
```

Comment:
Nope.  It's assigning into a _shared_ptr_ reference, so it needs to be a template (for shortest code).  It'll all go soon, we only have two more uses left of the deprecated version.